adb-utils.spec

```diff
@@ -31,14 +32,19 @@ specific service and directly including it to kickstart file.
 %{buildroot}%{_sysconfdir}/sysconfig/openshift_option
 %{__cp} services/openshift/scripts/* %{buildroot}/opt/adb/openshift/
 %{__cp} utils/* %{buildroot}/opt/adb/
+ln -s /opt/adb/sccli.sh %{buildroot}%{_bindir}/sccli
 
 %files
 %{_sysconfdir}/sysconfig/openshift_option
 %{_unitdir}/openshift.service
+%{_bindir}/sccli
```

Comment:
I think we should add sccli to /usr/local/bin as /usr/bin is distribution managed and /usr/local/bin is user managed